public_html/layouts/resources/app.js

```diff
@@ -903,45 +903,47 @@ app = {
 	},
 	updateRowHeight: function () {
 		var rowType = CONFIG.rowHeight;
-		if (rowType.length <= 0) {
-			//Need to update the row height
-			var widthType = app.cacheGet('widthType', 'mediumWidthType');
-			var serverWidth = widthType;
-			switch (serverWidth) {
-				case 'narrowWidthType' :
-					serverWidth = 'narrow';
-					break;
-				case 'wideWidthType' :
-					serverWidth = 'wide';
-					break;
-				default :
-					serverWidth = 'medium';
-			}
-			var userid = CONFIG.userId;
-			var params = {
-				'module': 'Users',
-				'action': 'SaveAjax',
-				'record': userid,
-				'value': serverWidth,
-				'field': 'rowheight'
-			};
-			AppConnector.request(params).then(function () {
-				$(rowType).val(serverWidth);
-			});
-		}
+		if (typeof rowType !== 'undefined') {
```

Comment:
rowType is always defined, wrong formatting below

Reply:
this erros happend in process installation

![firefox_2018-04-27_11-52-02](https://user-images.githubusercontent.com/18419851/39359304-6de11a60-4a11-11e8-9891-9b1993081f51.png)